work/flow.rst

```diff
@@ -503,7 +503,10 @@ If a rebase was required, a final check with Jenkins should be done.
 We encourage you to use this button when GitHub says "This branch has no conflicts with the base branch", "All checks have passed", and at least one of the checks has "Required" next to it, which should be the case for almost all repos that are being changed.
 For normally-configured repos, using the button will also delete the ticket branch after the merge.
 Do *not* select a different merge strategy from the pulldown next to the button; these should be disabled anyway.
+
 Also do *not* use the "Update my branch" button as that does a merge from main; rebase from the command line instead.
```

Comment:
It is okay to hit the little down arrow on the "Update my branch" button and select "Rebase" instead. I use that all the time. We probably should mention it. We can't make it the default unfortunately.

Reply:
I do too, but wasn't sure if I should include it explicitly in the dev guide.